### ISSUE TITLE
gotify ignores command line arguments / no way to redirect config

### ISSUE THREAD

**Issue by schneidersoft** [gotify/server#366]
Command line arguments given to gotify are ignored. There is f.i. no --help.
I was looking for a way to specify the config file to use. Gotify looks in ./gotify.yml and /etc/gotify... but not ~/.config/gotify/config.yml or ~/.gotify.yml which is what I was expecting...
There is also no way to specify where the data directory is located.
This means the behaviour of gotify depends on the current working dirctory...

**Comment by jmattheis** [comment]
Yeah, gotify either takes the config in /etc or the current working directory.

**Comment by hgy59** [comment]
> There is also no way to specify where the data directory is located.

I want to configure a custom data directory too, as the gotify server does not have permissions to create and write to a data directory in the bin folder (https://github.com/SynoCommunity/spksrc/pull/5126)
